app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#78DC94</color>
```

Comment:
should we extract this color?

Reply:
uh yea for sure, I just used the "Image asset" so it was left hard coded. I changed it to `safe_green`to match the splash screen